sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormPage.cs

```diff
@@ -6,6 +6,7 @@
 namespace Azure.AI.FormRecognizer.Models
 {
     /// <summary>
+    /// Represents a page recognized from the input document. Contains text lines, tables and page metadata.
```

Comment:
Based on the .NET readme, I thought you guys renamed `FormPage` to `RecognizedPage`

Reply:
I'm not sure if it was a typo. I couldn't find an issue with that ask, and we have it like `FormPage` in our ApiView. FYI I am changing that name in the reame in my samples PR: 
https://github.com/Azure/azure-sdk-for-net/pull/11491/
@annelo-msft  ?

Reply:
Yes, apologies, that was a bug in the README.  It is FormPage in the API we closed on with the team.